odl_video/settings.py

```diff
@@ -182,3 +182,27 @@
 VIDEO_S3_BUCKET = os.environ.get('VIDEO_S3_BUCKET', 'odl-video-service')
 VIDEO_S3_TRANSCODE_BUCKET = os.environ.get('VIDEO_S3_TRANSCODE_BUCKET', '{}_transcoded'.format(VIDEO_S3_BUCKET))
 VIDEO_S3_THUMBNAIL_BUCKET = os.environ.get('VIDEO_S3_THUMBNAIL_BUCKET', '{}_thumbnails'.format(VIDEO_S3_BUCKET))
+
+# AWS S3 upload settings
+# the defaults values come from the default configuration in boto3.s3.transfer.TransferConfig
+# apart from the first 2
+KB = 1024
+MB = KB * KB
+
+AWS_S3_UPLOAD_MULTIPART_THRESHOLD_MB = os.environ.get('AWS_S3_UPLOAD_MULTIPART_THRESHOLD_MB', 32)
```

Comment:
These environment variables will be interpreted as strings. Can you parse these into ints here and below, and into a bool at the end? Maybe we should use `get_var` from the cookiecutter here